sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CommonCoderTest.java

```diff
@@ -63,6 +63,7 @@
       .put("urn:beam:coders:bytes:0.1", ByteCoder.class)
       .put("urn:beam:coders:kv:0.1", KvCoder.class)
       .put("urn:beam:coders:varint:0.1", VarLongCoder.class)
+      .put("urn:beam:coders:stream:0.1", IterableCoder.class)
```

Comment:
This was already added by Luke. https://github.com/apache/beam/blob/d2b8b2886ce42f138b634d90208780bdce7e058e/sdks/python/apache_beam/coders/coders.py#L584 and
https://github.com/apache/beam/blob/13767de888d3f0d31cc482ce1f037b9b8fafb66e/sdks/java/core/src/main/java/org/apache/beam/sdk/coders/IterableCoder.java#L73